src/gui/gui_msgs.h

```diff
@@ -37,45 +37,62 @@ constexpr char help_msg[] =
 
 List of common options:
 
-  --printconf          Print the location of the default configuration file.
+  --printconf              Print the location of the default configuration file.
 
-  --editconf           Open the default configuration file in a text editor.
+  --editconf               Open the default configuration file in a text editor.
 
-  -c <command>         Run the specified DOS command before running FILE.
-                       Multiple commands can be specified.
+  --eraseconf              Delete the configuration file.
 
-  -noautoexec          Don't perform any [autoexec] actions.
+  --erasemapper            Delete the mapper file.
 
-  -noprimaryconf       Don't read settings from the primary configuration file
-                       located in your user folder.
+  --startmapper            Run the mapper GUI.
 
-  -nolocalconf         Don't read settings from "dosbox.conf" if present in
-                       the current working directory.
+  --securemode             Start in secure mode.
 
-  -conf <configfile>   Start with the options specified in <configfile>.
-                       Multiple configfiles can be specified.
+  --noautoexec             Ignore autoexec section in configuration file.
 
-  --working-dir <path> Set working directory to <path>. DOSBox will act as if
-                       started from this directory.
+  --opencaptures           Launches some external program (I don't even know...)
 
-  -fullscreen          Start in fullscreen mode.
+  --set <property>=<value> Sets a configuration property.
+                           Example: --set mididevice=fluidsynth
 
-  -lang <langfile>     Start with the language specified in <langfile>.
+  --socket <num>           Runs nullmodem on the specified socket number
```

Comment:
You're using imperative verbs consistently everywhere else (which is good), so this should be "run".

Reply:
Please put a period at the end of every config description.

Reply:
Fixed